kong/cmd/utils/prefix_handler.lua

```diff
@@ -509,6 +509,11 @@ local function prepare_prefix(kong_config, nginx_custom_template_path, skip_writ
     if not ok then
       return nil, err
     end
+
+    local success = pl_utils.executeex("chmod 755 " .. kong_config.socket_path)
+    if not success then
```

Comment:
```suggestion
    ok = pl_utils.executeex("chmod 755 " .. kong_config.socket_path)
    if not ok then
```
Also `executeex` returns `stderr`, should we handle it?

Reply:
done